packages/fluent-theme/src/fluent/styles/ChoiceGroupOption.styles.ts

```diff
@@ -86,6 +91,17 @@ export const ChoiceGroupOptionStyles = (props: IChoiceGroupOptionStyleProps): Pa
           borderColor: palette.neutralLighter
         }
     ],
+    innerField: [
```

Comment:
If you remove everything you adde and just put a 1px margin on `innerField` when it has Icon or Image you will reach the exact same result. Then add a comment to explain why it was needed and you are done. With porting to the default styles in v7.0 maybe it needs a little of redesign without breaking to much.